operational/flexera/custom_branding/README.md

```diff
@@ -0,0 +1,75 @@
+# Custom Branding
+
+## What It Does
+
+This Policy allows a customer to brand the FlexeraOne platform quickly and consistently.

```

Comment:
This should contain a bit more detail. It doesn't need to be a novel but 2-4 sentences with a bit more detail is usually expected.

"Policy" should not be capitalized. Generally, we should refer to policy templates as policy templates and not just as policies. I know this isn't consistently done right now but we should move in this direction to avoid confusion with other constructs like "applied policies"

Reply:
Unresolved because we're still referring to it as a Policy instead of as a policy template.